packages/asc-assets/package.json

```diff
@@ -20,7 +20,7 @@
     "assets": "assets"
   },
   "scripts": {
-    "build": "mkdir -p lib && cp -R src/* lib/"
+    "build": "mkdir -p lib && cp -R src/* lib/ && svgo --config=.svgo.yml -f ./src/Icons/ -o ./lib/Icons"
```

Comment:
Why not use SVGO in a Webpack loader?

Reply:
Because we don't create a bundle, we only provide optimized svg's in the package. It's up to the user of asc-assets (developer) how to bundle the code with the svg's.